editor.planx.uk/src/@planx/components/FindProperty/Public/Autocomplete.tsx

```diff
@@ -79,17 +79,37 @@ export default function PickOSAddress(props: PickOSAddressProps): FCReturn {
           latitude: selectedAddress.LAT,
           longitude: selectedAddress.LNG,
           organisation: selectedAddress.ORGANISATION || null,
-          sao: selectedAddress.SAO_TEXT,
+          sao: [
+            selectedAddress.SAO_START_NUMBER,
+            selectedAddress.SAO_START_SUFFIX,
+            selectedAddress.SAO_TEXT, // populated in cases of building name only, no street number
+          ]
+            .filter(Boolean)
+            .join(""),
```

Comment:
This updates existing `sao` field to be handled same as `pao`, whereas previously we only accounted for `SAO_TEXT` and would miss `SAO_START_NUMBER` or `SAO_START_SUFFIX`